src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/regexp/regexpsingleline/Example4.java

```diff
@@ -8,12 +8,11 @@
   </module>
 </module>
 */
+package com.puppycrawl.tools.checkstyle.checks.regexp.regexpsingleline;
 
 // xdoc section -- start
 /**
-* This file is copyrighted under CC. // Ok, as the file contains a copyright statement.
+* This file is copyrighted under CC.
```

Comment:
Please use in this config same `<property name="format" value="^[ ]*\*This file is copyrighted"/>`

Reply:
> Please use in this config same `<property name="format" value="^[ ]*\*This file is copyrighted"/>`

Done (slight tweak to the regex for the test to pass)